addons/mail/static/src/chatter/web/chatter_patch.js

```diff
@@ -192,6 +198,36 @@ patch(Chatter.prototype, {
         return _t("Unfollow");
     },
 
+    changeThreadIfNeeded(nextProps) {
+        const props = nextProps ? nextProps : this.props;
+        if (
+            this.props.threadId !== nextProps?.threadId ||
+            this.props.threadModel !== nextProps?.threadModel
+        ) {
+            this.state.thread.name = props.webRecord?.data?.display_name || undefined;
+            this.attachmentUploader.thread = this.state.thread;
+            if (props.threadId === false) {
+                if (this.state.thread.messages.length === 0) {
+                    this.state.thread.messages.push({
+                        id: this.messageService.getNextTemporaryId(),
+                        author: this.store.self,
+                        body: _t("Creating a new record..."),
+                        message_type: "notification",
+                        trackingValues: [],
+                        res_id: props.threadId,
+                        model: props.threadModel,
```

Comment:
Why not have that in portal too? It would make sense functionally and it would keep the code simpler too.

Reply:
The type of the fake message we send in creating a new record is `notification` and the patch of `message.xml` that demonstrates this kind of message is in the web bundle (actually there is no use case of message with notification type in portal) so there is an empty message on creating a record if I let this part remain.